www/base/index.html

```diff
@@ -36,9 +34,6 @@
     <noscript>You need to enable JavaScript to view this website.</noscript>
     <div id="root"></div>
   </body>
-  <script src="/browser-warning.js"></script>
```

Comment:
This one should be left as is because if browser is too old then it is not guaranteed that react code will be able to run at all.